std/os/time.zig

```diff
@@ -11,14 +11,16 @@ const posix = std.os.posix;
 pub const epoch = @import("epoch.zig");
 
 /// Sleep for the specified duration
-pub fn sleep(seconds: usize, nanoseconds: usize) void {
+pub fn sleep(nanoseconds: u64) void {
```

Comment:
Increased to a `u64` since 4 billion nanoseconds isn't all that long for 32-bit targets.